frontend/app/electron/main/tray-manager.ts

```diff
@@ -41,9 +42,23 @@ export class TrayManager {
         : []),
       { type: 'separator' },
       {
-        label: visible ? 'Minimize to tray' : 'Restore from tray',
-        click: () => this.toggleWindowVisibility(),
+        label: 'Display net worth on the tray',
+        type: 'checkbox',
+        checked: this.settings.appSettings.showNetWorthOnTray,
+        click: (item: MenuItem) => {
+          const showNetWorthOnTray = item.checked;
+          this.settings.appSettings.showNetWorthOnTray = showNetWorthOnTray;
+          this.settings.save();
+
+          this.update();
+        },
       },
+      ...(this.config.isMac
+        ? [{
+            label: visible ? 'Minimize to tray' : 'Restore from tray',
+            click: () => this.toggleWindowVisibility(),
+          }]
+        : []),
```

Comment:
why limit this only on mac? was it originally only on mac?

Reply:
The tray icon also works on linux and windows. Please don't remove it.

Reply:
LOL my bad, I placed the condition in the wrong spot. 
I wanted to set the condition for the new setting "Display net worth on the tray" to apply only on Mac and Linux, as Windows won't be showing it at all.

Reply:
Updated now @kelsos